src/main/java/org/fundacionjala/at15/katas/pokerhands/saul/HandRank.java

```diff
@@ -0,0 +1,238 @@
+package org.fundacionjala.at15.katas.pokerhands.saul;
+
+import static org.fundacionjala.at15.katas.pokerhands.saul.Constants.*;
+
+import java.util.List;
+
+public class HandRank {
```

Comment:
Ok, this class is really cool on separating the logic for checking the hand type, but it's getting too big. It can be improved by looking for a way to delegate the responsabilities or even create another hierarchy of objects that take care of finding the hand rank

Reply:
Hi Jose, I took your recommendations and I could refactor the project.